### DISCUSSION TITLE
Support footnotes

### DISCUSSION THREAD

**Discussion by KusakabeShi** [mkdocs/mkdocs#2875] [Q&A]
In this readme.md https://github.com/KSKBpage/dn42page/blob/main/README.md

# What I get: 
![image](https://user-images.githubusercontent.com/73118488/176840501-ee737f79-663a-4e54-aea3-b82c3e74a96d.png)
![image](https://user-images.githubusercontent.com/73118488/176840413-db13fc76-816a-4187-8cdb-a7e5bafb6268.png)

# What I expected:
![image](https://user-images.githubusercontent.com/73118488/147860424-cead6b1d-e2f9-49f3-9ad2-dc64db927fc4.png)
![image](https://user-images.githubusercontent.com/73118488/147860425-be97ec83-e430-4c7a-b85d-6e1987e467d6.png)

**Comment by KusakabeShi** [comment]
Found a extension can do this:
```
pip3 install pymdown-extensions
```
pydocs.yml
```
markdown_extensions:
  - pymdownx.tilde
  - pymdownx.extra
```

**Reply by Andre601** [reply]
The default python-markdown does have footnotes support through its `footnotes` extension.

Just add
```yaml
markdown_extensions:
- footnotes
```
...and you're good to go.

Docs: https://python-markdown.github.io/extensions/footnotes/

**Reply by facelessuser** [reply]
yep, `pymdownx.extra` is just including Python Markdown's footnotes extension
